test/unittests/test_triples.py

```
@@ -50,15 +50,253 @@ def test_load_plugin(self, load_plugin):
+        reasoner = ReversalReasoner()
+        input_triples = [Triple("Paris", "capital_of", "France")]
+        inferred = list(reasoner.infer(input_triples))
+        self.assertEqual(len(inferred), 1)
+        self.assertEqual(inferred[0].subject, "France")
+
+    def test_extract_and_infer(self):
+        from ovos_plugin_manager.templates.triples import TriplesReasoner, TriplesExtractor, Triple
+
+        class SimplExtractor(TriplesExtractor):
+            def extract_triples(self, documents):
+                for doc in documents:
```

Ruff (0.15.7)

[warning] Loop control variable `doc` not used within loop body

Rename unused `doc` to `_doc`

(B007)